crates/debug/src/transform/expression.rs

```diff
@@ -388,17 +459,40 @@ where
         }
         need_deref = frame_base.unwrap().need_deref;
     }
-    let base_len = parts.len();
     let mut code_chunk = Vec::new();
     macro_rules! flush_code_chunk {
         () => {
             if !code_chunk.is_empty() {
-                parts.push(CompiledExpressionPart::Code(code_chunk));
-                code_chunk = Vec::new();
+                push!(CompiledExpressionPart::Code(code_chunk.clone()));
+                code_chunk.clear()
```

Comment:
```suggestion
                parts.push(CompiledExpressionPart::Code(code_chunk));
                code_chunk = Vec::new();
```

Reply:
Can `push!(` stay? I was thinking about `code_chunk.clone()` and `code_chunk = Vec::new()` stuff.

Reply:
I hope to keep `push!(` (see above), but no idea how to fix the warning without artificially reading the `code_chunk` after the last macro invocation.

Reply:
I see. Yeah, it just needs `if !code_chunk.is_empty() {` without `code_chunk = Vec::new()`

Reply:
I did something that works. But I feel dense and am not sure if this is what you meant. Please feel free to fix.